vuln/npm/382.json

```diff
@@ -3,7 +3,7 @@
   "created_at": "2018-03-03",
   "updated_at": "2018-03-03",
   "title": "XSS in links",
-  "author": "bl4de (https://twitter.com/_bl4de)",
+  "author": "heyitsmeuralex",
```

Comment:
npm username is `nanalan` if you need that; or `<imalex.xyz>`.

Reply:
Would  you like

`nanalan (https://imalex.xyz)`

or 

`heyitsmeuralex (https://imalex.xyz)`

?

Reply:
The latter.